### DISCUSSION TITLE
Where can I get packages for manual system updates?

### DISCUSSION THREAD

**Discussion by IgorA100** [YetiForceCompany/YetiForce#56] [General]
There is an option in the menu to update from a file.
I have read a lot of information, but I could not find a link to download the package for manual system update anywhere

**Comment by rskrzypczak** [comment]
A list of available updates can always be found in `Software configuration → Logs → Updates`
if the system is registered correctly, you will be able to simply download it directly from the panel
check this: https://github.com/YetiForceCompany/YetiForce/issues/50#issuecomment-2453945192
